lib/eloop/patches/0003-Bump-timeout-parameters-from-int-to-long.patch

```diff
@@ -0,0 +1,87 @@
+From 03270444add875925fef1e2cb97647f45a394d30 Mon Sep 17 00:00:00 2001
+From: Alois Klink <alois@nquiringminds.com>
+Date: Tue, 6 Sep 2022 18:34:47 +0100
+Subject: [PATCH 3/3] Bump timeout parameters from int to long
+
+I'm not 100% sure why this was done.
+I'm guessing that on some platforms, the max int value is only
+32768, and that may cause overflow.
```

Comment:
Is this description correct?

I tried to understand what https://github.com/nqminds/edgesec/commit/00d465d8705fa439b52a513541dc03c1f4231e4d was doing, but I'm not 100% sure if I was right.